x-pack/plugins/security_solution/server/lib/entity_analytics/asset_criticality/asset_criticality_data_client.ts

```diff
@@ -256,10 +256,13 @@ export class AssetCriticalityDataClient {
       total: 0,
     };
 
-    let streamIndex = 0;
+    let streamIndex = 1; // It corresponds to the line number
```

Comment:
We also have a bulk upload JSON API, has this broken the error reporting for that? 

`x-pack/plugins/security_solution/server/lib/entity_analytics/asset_criticality/routes/bulk_upload.ts`

You may need to fix this "higher up"

Reply:
Could the UI not deal with translating zero indexes to line numbers? zero indexed seems like an acceptable stance for the API to take?